cmd/createFgaModel.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright © 2024 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	authz "github.com/canonical/identity-platform-login-ui/internal/authorization"
+	logging "github.com/canonical/identity-platform-login-ui/internal/logging"
+	monitoring "github.com/canonical/identity-platform-login-ui/internal/monitoring"
+	fga "github.com/canonical/identity-platform-login-ui/internal/openfga"
+	tracing "github.com/canonical/identity-platform-login-ui/internal/tracing"
+	"github.com/spf13/cobra"
+)
+
+// createFgaModelCmd represents the createFgaModel command
+var createFgaModelCmd = &cobra.Command{
+	Use:   "create-fga-model",
+	Short: "Creates an openfga model",
+	Long:  `Creates an openfga model`,
+	Run: func(cmd *cobra.Command, args []string) {
+		apiUrl, _ := cmd.Flags().GetString("api-url")
```

Comment:
`app create-fga-model --api-url ...` 
even though there is the `fga` bit, not sure `api-url` is enough to explain what url is needed
we might need to either change the name of the command and simplify the args or the other way round

anyway not a biggie